XDS_PROTOCOL.md

```diff
@@ -43,10 +43,10 @@ control of sequencing is required.
 Each xDS API is concerned with resources of a given type. There is a 1:1
 correspondence between an xDS API and a resource type. That is:
 
-* [LDS: `envoy.api.v2.Listener`](api/lds.proto)
-* [RDS: `envoy.api.v2.RouteConfiguration`](api/rds.proto)
-* [CDS: `envoy.api.v2.Cluster`](api/cds.proto)
-* [EDS: `envoy.api.v2.ClusterLoadAssignment`](api/eds.proto)
+* [LDS: `envoy.api.v2.Listener`](envoy/api/v2/lds.proto)
```

Comment:
Can you please also fix "discovery.proto" and "config_source.proto"  above? I think it makes more sense to point to master instead of an old SHA there.